### DISCUSSION TITLE
New core members

### DISCUSSION THREAD

**Discussion by yutingzhao1991** [ant-design/ant-design-web3#404] [General]
Seeking more [core members](https://web3.ant.design/guide/contributing#core-members), if you have substantial contributions to Ant Design Web3, we are happy to invite you to become a core member and participate together in the maintenance of the project.

If you are interested, you can leave us a message.

**Comment by yutingzhao1991** [comment]
@liangchaofei @gin-lsl Are you interested in this?

**Reply by gin-lsl** [reply]
of course, Thankssssssssss

**Reply by liangchaofei** [reply]
+1

**Comment by thinkasany** [comment]
I'm very interested in this project and have provided some contributions, can I join you as a core member? @yutingzhao1991 https://github.com/ant-design/ant-design-web3/commits?author=thinkasany

**Reply by yutingzhao1991** [reply]
We've seen that you've made many contributions, and we are delighted that you are willing to become a core member. Welcome aboard.
